src/External_Connectors.md

```diff
@@ -5,7 +5,7 @@
  Pin  | Name   | Explanation
 ------|--------|--------------
   1   | VDD    | Power Supply +5V DC
-  2   | PHI    | System Clock
+  2   | PHI    | System Clock ≅ 1.05 MHz <br> (CGB Double-speed mode: ≅ 2.10 MHz)
```

Comment:
I could find only one other usage of ≅for "approx equivalent to" but many more ~. Maybe this should be clarified document wise as well?

Reply:
Right - I looked for examples of shortened MHz values and duplicated from that one instance. I think it is useful to indicate that it's an approximated value vs the full one in some way.